spec/index.bs

```diff
@@ -4625,16 +4648,30 @@ interface mixin GPUProgrammablePassEncoder {
 
             **Arguments:**
             <pre class=argumentdef for="GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsetsData, dynamicOffsetsDataStart, dynamicOffsetsDataLength)">
-                index:
-                bindGroup:
-                dynamicOffsetsData:
-                dynamicOffsetsDataStart:
-                dynamicOffsetsDataLength:
+                |index|: The index to set the bind group at.
+                |bindGroup|: Bind group to use for subsequent render or compute commands.
+                |dynamicOffsetsData|: Array containing buffer offsets in bytes for each entry in
+                    |bindGroup| with marked as {{GPUBindGroupLayoutEntry/hasDynamicOffset}}.
```

Comment:
ok, here comes the fun part: what is the order?
In Vulkan, dynamic offsets need to be provided in the order of `binding`, which can be different from the order of either `BindGroupLayoutEntry` or `BindGroupEntry`. I suggest to follow this route and specify that the offsets are taken in the order of increased binding number

Reply:
Oof, I have no idea what's going to be best here. Gonna let the editors work this one out. 😅